Automattic/wp-calypso
# White glove: Modify product name in cart for the old checkout

#### Changes proposed in this Pull Request

This PR is a part of the larger PR that implements the white glove A/B test. Please check #41618 for more context.

* If a user clicks "Buy now" on the white glove offer page, then we add a Business plan in cart with the product name modified to indicate that it's a white glove product.
<img src="https://user-images.githubusercontent.com/1269602/83155016-6d1e1400-a11e-11ea-9b22-551ade0f4e09.png" />

* Completing the purchase will send a receipt to the email inbox with the name "white glove" mentioned in the receipt.

<img src="https://user-images.githubusercontent.com/1269602/83155499-ed447980-a11e-11ea-8358-5fa6ddf206b6.png" />
* This PR implements changes only for the old checkout. New checkout is handled in a separate PR.

* Rename "Quick Start" to "one-on-one" in the email that asks to schedule a concierge session

<img src="https://user-images.githubusercontent.com/1269602/83159763-f97f0580-a123-11ea-8257-0939186c6635.png" />

#### Testing instructions



**Note:** Make sure that you only see the old checkout for testing this PR. If you have a non-US IP, then there's nothing to do. If you have a US IP, then assign yourself to the `regular` variation fo the `showCompositeCheckout` A/B test.

Depends on D42624-code. Please apply patch in your sandbox.

1. Assign yourself to the `whiteGloveUpsell` test's variant and begin signup flow. Apply D42624-code to sandbox.
2. Select free plan and free domain, you should now see the offer page.
3. Click "No thanks" on the offer should take you to Customer Home
4. Click "Buy now" on the offer should take you to the checkout page.
5. Verify that the product name in cart says "WordPress.com Business (White glove edition)"
6. Complete purchase and check the receipt email sent to the user's inbox. Verify that the receipt mentions the product as "WordPress.com Business (White glove edition)" as shown in the screenshot in PR description.
